### ISSUE TITLE
Model compilation fails (undeclared identifier, gcc fails)

### ISSUE THREAD

**Issue by jvanhoefer** [AMICI-dev/AMICI#1263]
I tried to compile a simple SBML model via 

```python
import amici

model_name = 'Lotka_Volterra'
model_output_dir = 'Lotka_Volterra_AMICI'

sbml_importer = amici.SbmlImporter(sbml_file)
sbml_importer.sbml2amici(model_name,
                         model_output_dir)
```

The model is a valid SBML (tested via web validator, happy to share the model), COPASI produces the expected ODEs.

**What has happened instead?**
Error message: 

```
Lotka_Volterra_J.cpp:15:25: error: use of undeclared identifier 'y'
    J[0] = alpha - beta*y;
                        ^
1 error generated.
Testing SWIG executable swig4.0... FAILED.
Testing SWIG executable swig3.0... FAILED.
Testing SWIG executable swig... SUCCEEDED.
error: command 'gcc' failed with exit status 1
```

**AMICI version and system environment**
 Mac + AMICI 0.11.6

**Comment by dweindl** [comment]
@jvanhoefer : Can you provide the SBML model? What is `y` in your model?

**Comment by jvanhoefer** [comment]
a species, I can send it to you :)

**Comment by paulstapor** [comment]
Hi,
y is actually protected, since it's internally used for observables. Actually, this should be marked during import and code generation, but maybe we have a bug here...
Can you try with another name for this species which is not taken? Say, just using "a"? Or even better: "predator" and "prey"?

**Comment by jvanhoefer** [comment]
ah, that explains everything, wasn't aware, that I can not name a species `y`. I can and will just rename my species :) Thank you @paulstapor!

**Comment by paulstapor** [comment]
As said: In principle, it should be possible, because I think, AMICI should take care of those cases. However, this is at least a potential source of problems, so I would try to go via this first...

**Comment by dweindl** [comment]
Fixed in v0.11.7
